RecoTracker/PixelSeeding/plugins/CAHitNtupletGeneratorOnGPU.cc

```diff
@@ -197,6 +199,9 @@ void CAHitNtupletGeneratorOnGPU<pixelTopology::Phase1>::fillDescriptions(edm::Pa
   trackQualityCuts.add<double>("quadrupletMaxTip", 0.5)->setComment("Max |Tip| for quadruplets, in cm");
   trackQualityCuts.add<double>("quadrupletMaxZip", 12.)->setComment("Max |Zip| for quadruplets, in cm");
 
+  desc.add<int>("minYsizeB1", 1)->setComment("Min Y cluster size in pixel B1");
+  desc.add<int>("minYsizeB2", 1)->setComment("Min Y cluster size in pixel B2");
```

Comment:
@cms-sw/orp-l2, it would be great to get this PR into CMSSW_15_0_0 for 2025 HLT development purposes. 
As it is, it does change (HLT) physics, could it enter in pre3 still? 
Otherwise I guess it would more acceptable if it would just make the parameters configurable, followed by a change in the HLT menu configuration?

Reply:
Let's try to get it merged tomorrow, that will be fine for pre3

Reply:
excellent, thanks.